content/_releases/25/50.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Product Update - v25.06
+description: Details of the product update v25.06 from the 6th of February 2025.
+layout: rel
+version: "25.06"
+section: Product Updates
+category: product-updates
+releaseDate: 2024-02-06
+redirect_from:
+  - /releases/25.06.html
+---
+
+## New Features
+### New endpoint for retrieving sorted Workspaces list
```

Comment:
please add link to api docs https://api.elastic.io/docs/v2#/workspaces/get_workspaces_all_list